product/views/ManageIQ_Providers_MiddlewareManager.yaml

```diff
@@ -0,0 +1,83 @@
+#
+# This is an MIQ Report configuration file
+#   Single value parameters are specified as:
+#     single_value_parm: value
+#   Multiple value parameters are specified as:
+#     multi_value_parm:
+#       - value 1
+#       - value 2
+#
+
+# Report title
+title: Middleware Providers
+
+# Menu name
+name: EmsMiddleware
+
+# Main DB table report is based on
+db: ManageIQ::Providers::MiddlewareManager
+
+# Columns to fetch from the main table
+cols:
+- name
+- hostname
+- ipaddress
+- emstype_description
+- port
+
+# Included tables (joined, has_one, has_many) and columns
+include:
+  zone:
+    columns:
+    - name
+
+# Included tables and columns for query performance
+include_for_find:
+  :zone: {}
+  :tags: {}
```

Comment:
@martinpovolny Can you review this report...I can't remember the detail of reports, and I don't think zone and tags are need here, but maybe they are?